client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -191,6 +195,15 @@ class CancelPurchaseForm extends React.Component {
 		this.props.onInputChange( newState );
 	};
 
+	onSelectOneChange = ( option ) => {
+		const newState = {
+			...this.state,
+			questionOneText: option.value,
+		};
+		this.setState( newState );
```

Comment:
When updating the state by reading from a previous value of the state, I believe it's safer to do something like:
```
this.setState( prevState => {
    return {
        ...prevState,
        // ...
    };
);
```
If I remember well, it ensures you're working with the value that's expected, since setting the state is (or can be?) an asynchronous operation.

In this case you could probably do `this.setState( { questionOneText: option.value } );` though.